### ISSUE TITLE
sorting

### ISSUE THREAD

**Issue by sorawee** [sorawee/fmt#10]
There are stuff that might be able to sort safely. See e.g. https://github.com/russellw/racket-format/blob/master/sort.rkt (though a lot of stuff in there are actually not safe to sort). Perhaps provide an option to sort?

**Comment by jackfirth** [comment]
Are you thinking of things like `require` and `provide` specs?

**Comment by sorawee** [comment]
Yep. It's tricky though, especially when comments are involved. Perhaps only do it when there's no comment?

**Comment by jackfirth** [comment]
Yes, that sounds right.
